### PR TITLE
Recent usage warning

### PR DESCRIPTION
**Pull request by fredex42**
show warning if there are "recent" usages. "recent" is defined in imageUsageController

@clloyd @ShaunYearStrong

### CODE REVIEW

File: kahuna/public/js/components/gr-image-usage/gr-image-usage.js [hunk 1]
```diff
@@ -58,12 +58,19 @@ module.directive('grImageUsage', [function() {
...
+    'imageUsagesService',
+    function (imageUsagesService) {
     const ctrl = this;
 
     ctrl.formatTimestamp = (timestamp) => {
         return moment(timestamp).fromNow();
     };
+
+    ctrl.isRecent = (timestamp) => {
+        var nowtime = new Date();
```
Comment (clloyd) [inline kahuna/public/js/components/gr-image-usage/gr-image-usage.js]: This file seems to be using ES6 const/let. I think this should probably be `const` to match style

File: usage/app/lib/Config.scala [hunk 2]
```diff
@@ -85,7 +85,7 @@ object Config extends CommonPlayAppProperties with CommonPlayAppConfig {
       .withRegion(Regions.EU_WEST_1)
 
   val postfix = if (stage == "DEV") {
-    iamClient.getUser().getUser().getUserName()
+    "b0rked"
```
Comment (clloyd) [inline usage/app/lib/Config.scala]: Do we want to commit this?

File: kahuna/public/js/components/gr-image-usage/gr-image-usage.js [hunk 3]
```diff
@@ -58,12 +58,19 @@ module.directive('grImageUsage', [function() {
...
+        return moment(timestamp).isBetween(moment(nowtime).subtract(imageUsagesService.recentTime,'days'),moment(nowtime));
```
Comment (clloyd) [inline kahuna/public/js/components/gr-image-usage/gr-image-usage.js]: Can we use isAfter like below, I think it's easier to understand what's going on

File: kahuna/public/js/components/gr-image-usage/gr-image-usage.js (hunk 1 above)
Comment (kenoir) [inline kahuna/public/js/components/gr-image-usage/gr-image-usage.js]: :+1:

File: usage/app/lib/Config.scala (hunk 2 above)
Comment (kenoir) [inline usage/app/lib/Config.scala]: I think that was one of @Reettaphant's changes. Probably best to check with her what's going on here (if this isn't working for you and is the reason you've made this subsititution).

File: kahuna/public/config.js [hunk 4]
```diff
@@ -95,6 +95,10 @@ System.config({
     "github:jspm/nodelibs-util@0.1.0": {
       "util": "npm:util@0.10.3"
     },
+    "github:tapmodo/Jcrop@0.9.12": {
```
Comment (kenoir) [inline kahuna/public/config.js]: What's this for?

**Comment by kenoir** [comment]
Looks good apart from comments.

File: usage/app/lib/Config.scala (hunk 2 above)
Comment (fredex42) [inline usage/app/lib/Config.scala]: the reason i included it was that iamClient.getUser().getUser().getUserName() was causing an AWS permissions error when loading.  I agree that we could possibly change the string "b0rked" to something a little less irreverant, but I think we still need a static string value here unfortunately

Comment (fredex42) [inline usage/app/lib/Config.scala]: @Reettaphant do you have any preference?

File: kahuna/public/config.js (hunk 4 above)
Comment (fredex42) [inline kahuna/public/config.js]: I don't remember adding that manually, but I think it was a requirement somewhere along the line. I can remove it and see if it breaks :)

**Comment by fredex42** [comment]
gridwarning

**Comment by fredex42** [comment]
@kenoir - https://github.com/guardian/grid/pull/2011

**Pull request merged**